apps/client/src/components/QuestionAnswerTable.js

```diff
@@ -41,53 +41,59 @@ export default ({ checker, onGoToQuestion }) => {
-              <Question>{question.text}</Question>
-              {question.options ? (
-                <UserAnswer>{question.answer.replace(/['"]+/g, "")}</UserAnswer>
-              ) : (
-                <UserAnswer>
-                  {
-                    booleanOptions.find(
-                      (option) => option.value === question.answer
-                    ).label
-                  }
-                </UserAnswer>
-              )}
-              {onGoToQuestion && (
-                <Change>
-                  <Button
-                    onClick={() => onGoToQuestion(index)}
-                    variant="textButton"
-                  >
-                    Wijzig
-                  </Button>
-                </Change>
-              )}
-
-              {isDecisiveForPermits.map((permit, index) => (
-                <UserResult key={`${permit} ${index}`}>
-                  <Icon
-                    color={themeColor("secondary", "main")}
-                    size={30}
-                    style={{
-                      flexShrink: 0, // IE11 Fix
-                    }}
-                  >
-                    <Alert />
-                  </Icon>
-                  <UserResultParagraph strong>
-                    Op basis van dit antwoord bent u vergunningplichtig voor{" "}
-                    {permit.name.replace("Conclusie", "").toLowerCase()}
-                  </UserResultParagraph>
-                </UserResult>
-              ))}
-            </QuestionWrapper>
-          );
+            uniqBy(permitsPerQuestion[questionIndex], "name") || [];
+          if (
+            typeof question.answer === "boolean" ||
+            typeof question.answer === "string"
+          ) {
+            return (
+              <QuestionWrapper key={question.id}>
+                <Question>{question.text}</Question>
+                {question.options ? (
+                  <UserAnswer>
+                    {question.answer.replace(/['"]+/g, "")}
+                  </UserAnswer>
+                ) : (
+                  <UserAnswer>
+                    {
+                      booleanOptions.find(
+                        (option) => option.value === question.answer
+                      ).label
+                    }
+                  </UserAnswer>
+                )}
+                {onGoToQuestion && (
+                  <Change>
+                    <Button
+                      onClick={() => onGoToQuestion(questionIndex)}
+                      variant="textButton"
+                    >
+                      Wijzig
+                    </Button>
+                  </Change>
+                )}
+                {isDecisiveForPermits.map((permit, index) => (
+                  <UserResult key={`${permit} ${index}`}>
+                    <Icon
+                      color={themeColor("secondary", "main")}
+                      size={30}
+                      style={{
+                        flexShrink: 0, // IE11 Fix
```

Comment:
We should consider adding an atom for Icon if it's not safe to use without browser-hacks.
It's too easy to forget to implement such a 'fix'.
Not implemented in this change, I know.